Cargo.toml

```diff
@@ -51,6 +51,22 @@ bimap = { version = "0.6", features = ["serde"] }
 byteorder = "1"
 bytes = "1.1"
 chrono = "0.4"
+chrono-tz = "0.6"
+
+# Once a new version of clickhouse-rs is released, we can stop using a git
+# repository as a dependency. The declaration can then be replaced with the
+# following:
+#
+# clickhouse-rs = { version = "...", default-features = false, features = [
+#   "async_std",
+# ] }
+#
+# Releases can be tracked at [1].
+#
+# [1]: https://crates.io/crates/clickhouse-rs/versions
+clickhouse-rs = { git = "https://github.com/suharev7/clickhouse-rs", branch = "async-await", default-features = false, features = [
```

Comment:
should we pin this to a ref? We don't control the fork so a branch could break the build at a random time if it gets updated in a unpleasant way.

Reply:
As can be seen in the [`Cargo.lock`](https://github.com/scrabsha/tremor-runtime/blob/a3e16a2585b9963a0a46afd9e27dbe0b582d6f38/Cargo.lock#L1243) file, cargo defaulted it to the last commit of the `async-await` branch at the time I added the dependency. As a result, it won't "automatically update" if a commit is pushed in this branch.

From [the cargo references](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories):
> Once a git dependency has been added, Cargo will lock that dependency to the latest commit at the time. New commits will not be pulled down automatically once the lock is in place. However, they can be pulled down manually with cargo update.

That being said, I would totally understand if you prefer having a specific `rev` instead of a `branch` here :)

Reply:
Oh, awesome :) learned something new! I think a `rev` might still be slightly better since I know at times we end up with a lock file so messed up from a merge :sob: that we delete and re-create it.

Reply:
That's a very fair point. It should be fixed by commit b5082120.